lib/controllers/rocks.js

```diff
@@ -0,0 +1,26 @@
+const { Router } = require('express');
+const Rock = require('../models/Rock');
+module.exports = Router()
+.patch('/:id', async (req, res) => {
```

Comment:
We might organize these with our post first, then our gets, then our patch and finally our delete